waiter/test/waiter/scheduler/kubernetes_test.clj

```diff
@@ -394,11 +394,30 @@
                              :annotations {:waiter/port-count "1"
                                            :waiter/service-id "test-app-1234"}}
                   :spec {:containers [{:ports [{:containerPort 8080 :protocol "TCP"}]}]}
-                  :status {:podIP "10.141.141.11"
+                  :status {:phase "Pending"
+                           :podIP "0.0.0.0"
```

Comment:
Why add the `:podIP` field here? I think Kubernetes leaves this key out if an IP hasn't been assigned yet.

Reply:
This was added for the tests as `live-pod?` filters out entries that do not have IPs. That will be fixed in the follow-up PR addressing container creating.